Makefile

```diff
@@ -82,3 +82,11 @@ ui-up:
 
 ui-down:
 	docker-compose down
+
+ui-build:
+	npm --prefix web/client ci && npm --prefix web/client run build
```

Comment:
to run `npm` as part of `Makefile` we need to have nodejs installed globally

Reply:
It should work during release but if just run `make ui-build` locally without `node` , `npm` won't be there.

Reply:
Yeah I think that is fine the person would get an error. This is something that only people with access to tobiko data creds can do.

Reply:
I understand now and didn't realize that even when doing UI work people were building from within docker. Updated: https://github.com/TobikoData/sqlmesh/pull/1331/commits/fec490a23d770982ed57a6de8ef7015b57dc7fef